package.json

```diff
@@ -33,12 +37,14 @@
         "react-select": "^5.2.1",
         "sha.js": "^2.4.11",
         "stream-browserify": "^3.0.0",
+        "stream-http": "^3.2.0",
         "text-encoding": "^0.7.0",
+        "url": "^0.11.0",
         "uuid": "^8.3.2",
         "web-vitals": "^2.1.4"
     },
     "scripts": {
-        "start": "craco start",
+        "start": "GENERATE_SOURCEMAP=false craco start",
```

Comment:
What's the purpose of this?

Reply:
> What's the purpose of this?

Avoid a big bunch of warnings (`Failed to parse source map`...)